cli/pkg/config_manager/config_manager.go

```diff
@@ -0,0 +1,489 @@
+		prompt2 := &survey.Select{
+			Message: fmt.Sprintf("What kind of service is %s?", serviceName),
+			Options: []string{
+				serviceTypeExternal,
+				serviceTypeInternal,
+				serviceTypePrivate,
+			},
+			Default: serviceType,
+		}
+
+		err = survey.AskOne(prompt2, &serviceType)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to obtain an aws profile")
+		}
+
+		var ok bool
+		serviceType, ok = serviceTypeMapping[serviceType]
+		if !ok {
+			return nil, errors.Wrapf(err, "unexpected service type")
+		}
+
+		port := ""
+		prompt3 := &survey.Input{
+			Message: fmt.Sprintf("Which port does service %s listen on?", serviceName),
+			Default: defaultServicePort,
+		}
+		err = survey.AskOne(prompt3, &port,
+			survey.WithValidator(survey.Required),
+			survey.WithValidator(survey.MinLength(2)),
+			survey.WithValidator(survey.MaxLength(5)),
+			survey.WithValidator(Port))
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to prompt")
+		}
+
+		uri := ""
+		prompt3 = &survey.Input{
+			Message: fmt.Sprintf("Which uri does %s respond on?", serviceName),
+			Help:    "This is the relative path that the service will respond on, e.g. /api/v1",
+			Default: "/",
+		}
+		err = survey.AskOne(prompt3, &uri,
+			survey.WithValidator(survey.Required),
+			survey.WithValidator(survey.MinLength(1)),
+			survey.WithValidator(survey.MaxLength(255)),
+			survey.WithValidator(URI))
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to prompt")
+		}
+
+		uri, _ = strings.CutSuffix(uri, "/")
+		portNumber, err := strconv.Atoi(port)
+
+		if err != nil {
+			return nil, errors.Wrap(err, "port number is not valid")
+		}
+
+		services = append(services, Service{
+			Name:           serviceName,
+			ServiceType:    serviceType,
+			DockerfilePath: dockerFileName,
+			Context:        contextPath,
+			Port:           portNumber,
+			Uri:            uri,
+		})
+	}
+
+	if len(services) == 0 {
+		return nil, errors.New("you have not configured any services")
+	}
+
+	logrus.Info("Done checking for services")
+
+	happyConfig.GetData().Environments = environments
+	happyConfig.GetData().FeatureFlags = config.Features{
+		EnableDynamoLocking:   true,
+		EnableHappyApiUsage:   true,
+		EnableECRAutoCreation: true,
+	}
+	happyConfig.GetData().DefaultEnv = environmentNames[0]
+	happyConfig.GetData().DefaultComposeEnvFile = ".env.ecr"
+
+	serviceDefs := map[string]any{}
+	stackDefaults := map[string]any{
+		"stack_defaults":   "git@github.com:chanzuckerberg/happy//terraform/modules/happy-stack-eks?ref=main",
+		"routing_method":   "CONTEXT",
+		"create_dashboard": false,
+		"app":              appName,
+		"services":         serviceDefs,
+	}
+
+	serviceNames := []string{}
+	// sort services by length of service.Uri in reverse order
+	sort.Slice(services, func(i, j int) bool {
+		return len(services[i].Uri) > len(services[j].Uri)
+	})
+	priority := 0
+	for _, service := range services {
+		service.Priority = priority
+		priority++
+	}
```

Comment:
this is a deprecated field, so I don't think we need to worry about it

Reply:
No harm in populating it since it is still a valid argument.